bindings/python/tox.ini

```diff
@@ -29,7 +29,6 @@ deps =
     -r requirements-dev.txt
     pylint
 commands =
-    pip install .
```

Comment:
Why can this be removed? If I run `tox -e lint-sdk` this wouldn't use the latest changes?

Reply:
Because it checks the source folder which doesn't get modified by installation, for tests and examples we need to build it, because there the sdk is imported